client/constants/index.js

```diff
@@ -39,6 +39,15 @@ export const GENERIC_ITEM_ACTIONS = {
         label: 'Spike',
         icon: 'icon-trash',
     },
+    UNSPIKE: {
+        label: 'Unspike',
+        icon: 'icon-unspike',
+    },
+    DUPLICATE: {
+        label: 'Duplicate',
+        icon: 'icon-copy',
+    },
+    HISTORY: { label: 'View History' },
```

Comment:
@fritzSF , as we don't have an icon for item history, we thought of using the same we use for version history in Superdesk (big-icon--history). But we don't have the small sized counterpart of that icon. Could you please advise ?